hww8/hh8.py

```diff
@@ -0,0 +1,60 @@
+# Быки и коровы
+import random
+
+numbers = list(range(10))
+random.shuffle(numbers)
+
+comp_number = ""
+for i in set(numbers[:4]):
+    comp_number += str(i)
+
+print("Computer made the 4 digit number, try to guess it!")
+
+bulls = 0
+while bulls != 4:
+    guess_number = input("Enter 4 digit number with different digits: ")
+    if (len(guess_number) != 4 or not guess_number.isdigit() or
+            len(set(guess_number)) != 4):
+        print("Enter valid 4 digit number")
+        continue
+
+    bulls = 0
+    cows = 0
+    index = 0
+
+    for x in guess_number:
+        if x == comp_number[index]:
+            bulls += 1
+        elif x in comp_number:
+            cows += 1
+        index += 1
```

Comment:
You do not need to introduce an index:
```suggestion
    for x in range(len(guess_number)):
        if guess_number[x] == comp_number[x]:
            bulls += 1
        elif guess_number[x] in comp_number:
            cows += 1
```

Reply:
fixed